src/components/FloatingActionButton.js

```diff
@@ -43,7 +45,7 @@ export default class FloatingActionButton extends Component {
             transition: 'max-width 0.5s ease-out, padding 0.1s linear 0.2s'
           }
         }, label),
-        icon(iconShape, { size: 25, style: { stroke: 'white', strokeWidth: 2 } })
+        icon(iconShape, { size: 25, style: { stroke: 'white', strokeWidth: 0.25 } })
```

Comment:
I initially had made it thinner when I was trying to use the three-dot card menu since the wide thickness looked weird with that icon. However, since we're doing the two buttons for now instead, I'll change it back since it looks fine both ways!